### PR TITLE
app-misc/rdfind: New package, import from ::guru

### PR DESCRIPTION
**Pull request by tastytea**
Find duplicate files based on their content

Closes: https://bugs.gentoo.org/637384
Signed-off-by: Ronny (tastytea) Gutbrod <gentoo@tastytea.de>

### CODE REVIEW

File: app-misc/rdfind/metadata.xml
Comment (tastytea) [reply app-misc/rdfind/metadata.xml]: Fixed/added everything except ~arm.

File: app-misc/rdfind/rdfind-1.4.1.ebuild
Comment (juippis) [reply app-misc/rdfind/rdfind-1.4.1.ebuild:15]: Everything else looks good, but I can't merge this while adding the `~arm`. We've got policies for that. So please remove the KEYWORD and let's request it correctly from bugzilla after this has been merged.
https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages

Comment (tastytea) [reply app-misc/rdfind/rdfind-1.4.1.ebuild]: Oh sorry, I remembered the manual differently. 😄 Fixed.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**
